Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.yml

```diff
@@ -0,0 +1,1824 @@
+  defaultvalue: https://127.0.0.1
+  display: Server URL (i.e., https://host:port)
+  name: url
+  required: true
+  type: 0
+- additionalinfo: OAuth Client ID and Client Secret to authorize third-party applications to communicate with Symantec EDR.
+  display: Client ID
+  name: credentials
+  displaypassword: Client Secret
+  required: true
+  type: 9
+- defaultvalue: 'false'
+  display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- additionalinfo: Retrieve incident related events from EDR database. An additional API call will be made for each fetched incident.
+  display: Fetch incidents alerts
+  name: isIncidentsEvent
+  required: false
+  type: 8
+- additionalinfo: Retrieve incident comments for each fetched incident when checked. An additional API call will be made for each fetched incident.
+  display: Fetch incident comments
+  name: isIncidentComment
+  required: false
+  type: 8
+- additionalinfo: If not selected, will fetch new incidents.
+  defaultvalue: Open
+  display: Status to filter out fetching as incidents. Comma-separated lists are supported, e.g., New,In-Progress
+  name: fetch_status
+  options:
+  - Open
+  - In-Progress
+  - Waiting
+  - Closed
+  required: false
+  type: 16
+- additionalinfo: If not selected, will fetch high and medium incidents.
+  defaultvalue: High,Medium
+  display: Priority to filter out fetching as incidents. Comma-separated lists are supported, e.g., Medium,High.
+  name: fetch_priority
+  options:
+  - Low
+  - Medium
+  - High
+  required: false
+  type: 16
+- additionalinfo: First Fetch timestamp. Default is 3 days. The maximum time range is 30 days. For example, if configured as 60 days based on the current datetime, then data will be fetched according to the time range using start_time=60 days and end_time=30 days.
+  defaultvalue: '3 days'
+  display: First fetch timestamp (<number> <time unit>, e.g., 10 minutes, 12 hours, 7 days)
+  name: first_fetch
+  required: false
+  type: 0
+- defaultvalue: '50'
+  display: Maximum number of incidents to fetch
+  name: fetch_limit
+  required: false
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+description: Symantec EDR (On Prem) endpoints help to detect threats in your network by filter endpoints data to find Indicators of Compromise (IoCs) and take actions to remediate the threat(s). EDR on-premise capabilities allow incident responders to quickly search, identify, and contain all impacted endpoints while investigating threats using a choice of on-premises.
+display: Symantec Endpoint Detection and Response (EDR) - On Prem
+name: SymantecEDRStage
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: The device ID of the target computer.
```

Comment:
Should this description be: A comma-separated list of device IDs of the target computers.